server/oauth-google.js

```diff
@@ -1,12 +1,8 @@
-var google = require("googleapis");
+//var google = require("googleapis");
```

Comment:
Not that it's any kind of problem, but you can just go ahead and confidently delete this line